Makefile

```diff
@@ -313,11 +320,20 @@ host-plugin-release:
 	@echo dev: need docker login with user in contiv org
 	docker plugin push ${CONTIV_V2PLUGIN_NAME}
 
-only-tar:
-
-tar: clean-tar
-	CONTIV_NODES=1 ${MAKE} build
-	@tar -jcf $(TAR_FILE) -C $(GOPATH)/src/github.com/contiv/netplugin/bin netplugin netmaster netctl contivk8s netcontiv -C $(GOPATH)/src/github.com/contiv/netplugin/scripts contrib/completion/bash/netctl -C $(GOPATH)/src/github.com/contiv/netplugin/scripts get-contiv-diags
+##########################
+## Packaging and Releasing
+##########################
+
+# build tarball
+tar: compile-with-docker
+	docker run --rm netplugin-build:$(NETPLUGIN_CONTAINER_TAG) \
+		-C /go/src/github.com/contiv/netplugin netplugin-version \
+	| tar -x netplugin-version
```

Comment:
Please use docker cp here. Piping can break if there's any Docker regression.

Reply:
From the docs: `docker cp [OPTIONS] CONTAINER:SRC_PATH DEST_PATH|-`

At this point there has never been a container, only an image, so there is no way I could use docker cp.

Reply:
@chrisplo: You can use `docker create` to create a container without running. You can remove it with `docker rm`.

Relying on the stdout isn't recommended. There have been regressions in the past with relying on piping from stdout and piping via stdin.

Reply:
cool, I'll update with create/cp

Reply:
actually did this in #1008, and removed this section as we don't depend on it for this PR